client/src/main/java/com/battleships/Client.java

```diff
@@ -1,23 +1,28 @@
 package com.battleships;
 
+import com.battleships.windowScalling.WindowScalling;
 import javafx.application.Application;
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
 public class Client extends Application {
+    private WindowScalling windowScalling;
+
+    public static void main(String[] args) {
+        launch(args);
+    }
 
     @Override
     public void start(Stage primaryStage) throws Exception {
```

Comment:
it is growing. You can divide it in separate methods

Reply:
class refactored